Launching-new-features.md

```diff
@@ -322,11 +254,6 @@ Note: since only users with release-coordinator permission can edit the settings
 
 We use the following principles to determine the value of a feature flags:
 
-- Each feature flag has multiple *rules*, its value changes to the value specified in the first rule that is matched. If no rule is matched, its value remains default.
-- Each rule has multiple *filters*, and a rule is matched only when all of its filters are matched.
-- Each filter describes a scenario. There are multiple types of filters (like `platform type` and `app version`) which can be used in combination with each other to describe various scenarios (like `platform type = Web OR platform type = Android`).
-- Each filter can have multiple conditions, and a filter is matched if any of the conditions specified in the rule are matched.
-
-For example, if we want a feature flag to be enabled on Web, we can configure the flag's rules/filters as shown below.
+- Feature flag can be enabled via 2 methods, by editing `Rollout percentage for logged-in users (0-100)` or `Force-enable for all users`. The rollout-percentage method is use to enable the feature-flag for the defined percentage of logged-in users. The force-enable method is use to force enable the feature flag for logged-in as well as the logged-out users. Setting force enable property will disable the effect of all the existing properties.
```

Comment:
I suggest you change all of this (including the "We use ..." part above) to:

Feature flags can be configured in two ways:
- **Editing 'Rollout percentage for logged-in users (0-100)'.** This turns on the feature flag for the specified percentage of logged-in users. Logged-out users are not affected.
- **Editing 'Force-enabled for all users'.** This turns on the feature flag for all logged-in and logged-out users, and overrides all the other feature flag settings.

Reply:
Done